auslib/db.py

```diff
@@ -2537,12 +2537,23 @@ class AUSDatabase(object):
     engine = None
     migrate_repo = path.join(path.dirname(__file__), "migrate")
 
-    def __init__(self, dburi=None, mysql_traditional_mode=False):
+    def __init__(self, dburi=None, mysql_traditional_mode=False, notify_tables=None):
         """Create a new AUSDatabase. Before this object is useful, dburi must be
            set, either through the constructor or setDburi()"""
         if dburi:
             self.setDburi(dburi, mysql_traditional_mode)
         self.log = logging.getLogger(self.__class__.__name__)
+        if notify_tables:
+            self.notify_tables = notify_tables
```

Comment:
This works because in practice, AUSDatabase is always constructed with dburi being passed, via https://github.com/mozilla/balrog/blob/4d6f673a850cbd2924eb55145f047fadfceabeea/auslib/global_state.py#L18.

However, it's quite clear that if someone were to construct an AUSDatabase without dburi, this would fail. I didn't realize this when I was writing it - I'll fix it up.